spec.html

```diff
@@ -26311,7 +26271,7 @@ <h1>Object.preventExtensions ( _O_ )</h1>
 
       <emu-clause id="sec-object.prototype">
         <h1>Object.prototype</h1>
-        <p>The initial value of `Object.prototype` is %Object.prototype%.</p>
+        <p>The initial value of `Object.prototype` is the Object prototype object.</p>
```

Comment:
This and many of the subsequent changes seem unrelated to this PR; is this a rebase conflict? If it's intentional, I'd ask that these changes go into their own PR.

Reply:
It was intentional to make everything consistent. I can pull them out into another PR tho.

Reply:
I believe that it should probably be done in its own PR/commit.

Reply:
@bakkot i extracted it to #2131 and rebased this one on top of it, so you can review just the commit for this PR separately. If you'd prefer to wait to review this one until after #2131 lands, that's fine too.